core/db/migrate/20161029130019_add_address_format_settings_to_countries.rb

```diff
@@ -0,0 +1,9 @@
+class AddAddressFormatSettingsToCountries < ActiveRecord::Migration[5.0]
+  def change
+    add_column :spree_countries, :provence_label, :string
+    add_column :spree_countries, :provence_specified, :integer, limit: 1
```

Comment:
note: this field is to fully replace ```states_required```; a proper clean-up will need to be implemented